frontend/viewdir/responsabilities.njk

```diff
@@ -0,0 +1,2 @@
+{% extends "template.njk" %}
```

Comment:
Is this file used or can it be removed?

Reply:
Also if the page is needed, can it be renamed to "responsibilities.njk"